README.md

```diff
@@ -0,0 +1,80 @@
+# Cucumber.ml
+
+This implements the core Cucumber feature file language, Gherkin, and
+assocated library for specifying the execution of those tests for the
+OCaml programming language.
+
+WARNING: This is still under heavy development and is provided as-is.
+This is for the adventurous who do not mind rough edges and
+half-implemented features.  All pull-requests are gratefully accepted.
+
+## libgherkin.so
+
+To be able to run the code, you will need to have compiled and
+installed the Gherkin library as a shared object in your Linux
+distribution (so that it will be avaliable to the linker at run time).
+You can do this by checking out the Cucumber project from Github
+[Cucumber](https://github.com/cucumber/cucumber) and applying pull
+request number [321](https://github.com/cucumber/cucumber/pull/321)
+then compiling the .so file and installing it in /usr/lib (or
+equivalent for your OS).  To compile the test runner (more below), you
+will need to install the .h files in /usr/include (or equivalent for
+your OS).
+
```

Comment:
Should we provide an `install` target for this? Isn't that the common way to do this?